connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -674,10 +695,7 @@ ble_error_t Gap::updateConnectionParameters(
         maxConnectionEventLength.value()
     );
 }
-#endif
-
 
-#if BLE_FEATURE_CONNECTABLE
```

Comment:
Please reintroduce the `BLE_FEATURE_CONNECTABLE` block, this for both slave and master. Not just central. 
see BLUETOOTH SPECIFICATION Version 5.0 | Vol 6, Part D Section 6.12

Reply:
this is not removed, it's just in a block